locale/en/docs/es6.md

```diff
@@ -9,7 +9,7 @@ Node.js is built against modern versions of [V8](https://developers.google.com/v
 All ECMAScript 2015 (ES6) features are split into three groups for **shipping**, **staged**, and **in progress** features:
 
 * All **shipping** features, which V8 considers stable, are turned **on by default on Node.js** and do **NOT** require any kind of runtime flag.
-* **Staged** features, which are almost-completed features that are not considered stable by the V8 team, require a runtime flag: `--es_staging` (or its synonym, `--harmony`).
+* **Staged** features, which are almost-completed features that are not considered stable by the V8 team, require a runtime flag: `--es_staging` (or its synonym, `--harmony`). Note: these flags are exposed by v8 and will potentially change without any deprecation notice.
```

Comment:
I think you want this down in **in progress** I believe this is for the generic `--harmony` flag